Admin.js

```diff
@@ -23,11 +23,8 @@ var Admin = {
         Admin.log("[core|shared_setup] Register services on", subject);
         Admin.set_object_field_value(subject);
```

Comment:
- eslint: Unquoted property 'height' found.
- eslint: Extra space before value for key 'height'.
- eslint: Strings must use doublequote.